### PR TITLE
tools: update doc for pd-recover

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Update doc for pd-recover.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-4.0**, **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/2410
- Other reference link(s):

### CODE REVIEW

File: pd-recover.md [hunk 1]
```diff
@@ -6,42 +6,152 @@ category: reference
...
+
+```
+10.0.1.15 | CHANGED | rc=0 >>
+[2019/10/14 07:06:35.278 +00:00] [INFO] [tikv-server.rs:464] ["connect to PD cluster 6747551640615446306"]
+……
+```
+
+### Get Alloc ID (TiKV StoreID)
+
+When you specify `alloc-id`, you need to specify a value greater than the currently largest `Alloc ID`. You can either use `ansible ad-hoc` in the Control Machine, or view the log directly on the server.
```
Comment (TomShawn) [inline pd-recover.md:69]: @shafreeck Please check whether my comment is technically correct, thanks!
```suggestion
The `alloc-id` value you specify must be larger than the currently largest `Alloc ID` value. To get `Alloc ID`, you can either use the `ansible ad-hoc` command in the Control Machine, or view the log directly on the server.
```

File: pd-recover.md [hunk 2]
```diff
@@ -6,42 +6,152 @@ category: reference
...
+
+#### Get [info] allocates id from PD log
```
Comment (TomShawn) [inline pd-recover.md:71]: @ran-huang Please wrap id names in `backquotes` in this document.
```suggestion
#### Get `[info] allocates id` from PD log
```

File: pd-recover.md [hunk 3]
```diff
@@ -6,42 +6,152 @@ category: reference
...
+{{< copyable "shell-regular" >}}
+
+```
+ansible-playbook rolling_update.yml --tags=tidb,tikv
+```
+
+## FAQ
 
-### Recovery flow
+### Multiple cluster IDs are found when you get the cluster ID
```
Comment (TomShawn) [inline pd-recover.md:151]:
```suggestion
### Multiple cluster IDs are found when getting the cluster ID
```

File: pd-recover.md [hunk 4]
```diff
@@ -6,42 +6,152 @@ category: reference
...
 
-1. Obtain the Cluster ID and the Alloc ID from the current cluster.
+When a PD cluster is created, a new cluster ID is generated. You can determine the cluster ID of the old cluster by viewing the log.
 
-     - Obtain the Cluster ID from the PD, TiKV and TiDB log.
-     - Obtain the allocated Alloc ID from either the PD log or the `Metadata Information` in the PD monitoring panel.
+### An error `dial tcp 10.0.1.13:2379: connect: connection refused` is returned when you execute pd-recover
```
Comment (TomShawn) [inline pd-recover.md:155]:
```suggestion
### The error `dial tcp 10.0.1.13:2379: connect: connection refused` is returned when executing `pd-recover`
```

File: pd-recover.md [hunk 5]
```diff
@@ -6,42 +6,152 @@ category: reference
...
 
-     Specifying `alloc-id` requires a number larger than the current largest Alloc ID. If you fail to obtain the Alloc ID, you can make an estimate of a larger number according to the number of Regions and Stores in the cluster. Generally, you can specify a number that is several orders of magnitude larger.
-2. Stop the whole cluster, clear the PD data directory, and restart the PD cluster.
-3. Use PD Recover to recover and make sure that you use the correct `cluster-id` and appropriate `alloc-id`.
-4. When the recovery success information is prompted, restart the whole cluster.
+The PD service is required when you execute pd-recover. Deploy and start the PD cluster before you use pd-recover.
```
Comment (TomShawn) [inline pd-recover.md:157]: `pd-recover` is the command name while PD Recover is the tool name. 
```suggestion
The PD service is required when you execute `pd-recover`. Deploy and start the PD cluster before you use PD Recover.
```

File: pd-recover.md [hunk 6]
```diff
@@ -6,42 +6,152 @@ category: reference
...
 
-1. [Go](https://golang.org/) Version 1.9 or later
-2. In the root directory of the [PD project](https://github.com/pingcap/pd), use the `make` command to compile and generate `bin/pd-recover`
+This section describes how to use PD Recover to recover a PD cluster.
 
-## Usage
+### Get cluster ID
 
-This section describes how to recover a PD cluster which cannot start or provide services normally.
+The cluster ID can be obtained in the log of PD, TiKV or TiDB. You can either use `ansible ad-hoc` in the Control Machine, or view the log directly on the server.
```
[the remark this answers is not in the document]
Comment (shafreeck) [reply pd-recover.md:17]: LGTM

File: pd-recover.md (hunk 1 above)
Comment (shafreeck) [reply pd-recover.md:69]: LGTM

**Comment by ran-huang** [comment]
@TomShawn PTAL again. Thanks~

File: pd-recover.md [hunk 7]
```diff
@@ -7,42 +7,152 @@ aliases: ['/docs/dev/reference/tools/pd-recover/']
...
+……
+```
+
+### Get `Alloc ID` (TiKV StoreID)
+
+The `alloc-id` value you specify must be larger than the currently largest `Alloc ID` value. To get `Alloc ID`, you can either use the `ansible ad-hoc` command in the Control Machine, or view the log directly on the server.
+
+#### Get `[info] allocates id` from PD log
+
+To get the `[info]` allocates id from the PD log, run the following command:
```
Comment (TomShawn) [inline pd-recover.md:74]:
```suggestion
To get the `[info] allocates id` from the PD log, run the following command:
```

File: pd-recover.md [hunk 8]
```diff
@@ -7,42 +7,152 @@ aliases: ['/docs/dev/reference/tools/pd-recover/']
...
--key string
-      Specify the path to the SSL certificate key file in PEM format, which is the private key of the certificate specified by `--cert`
--cluster-id uint
-      Specify the Cluster ID of the original cluster
--endpoints string
-      Specify the PD address (default: "http://127.0.0.1:2379")
+
+#### Get `[info] alloc store id` from TiKV log
+
+To get the `[info]` alloc store id from the TiKV log, run the following command:
```
Comment (TomShawn) [inline pd-recover.md:91]:
```suggestion
To get the `[info] alloc store id` from the TiKV log, run the following command:
```

File: pd-recover.md [hunk 9]
```diff
@@ -7,42 +7,152 @@ aliases: ['/docs/dev/reference/tools/pd-recover/']
...
-## Usage
+### Get cluster ID
 
-This section describes how to recover a PD cluster which cannot start or provide services normally.
+The cluster ID can be obtained from the log of PD, TiKV or TiDB. To get the cluster ID, you can either use the `ansible ad-hoc` command in the Control Machine, or view the log directly on the server.
 
-### Flags description
+#### Get `[info] cluster ID` from PD log (recommended)
 
+To get the `[info]` cluster ID from the PD log, run the following command:
```
Comment (TomShawn) [inline pd-recover.md:22]:
```suggestion
To get the `[info] cluster id` from the PD log, run the following command:
```

File: pd-recover.md [hunk 10]
```diff
@@ -7,42 +7,152 @@ aliases: ['/docs/dev/reference/tools/pd-recover/']
...
+
+```
+10.0.1.13 | CHANGED | rc=0 >>
+[2019/10/14 10:35:38.880 +00:00] [INFO] [server.go:212] ["init cluster id"] [cluster-id=6747551640615446306]
+……
+```
+
+#### Get `[info] cluster ID` from TiDB log
+
+To get the `[info]` cluster ID from the TiDB log, run the following command:
```
Comment (TomShawn) [inline pd-recover.md:38]:
```suggestion
To get the `[info] cluster ID` from the TiDB log, run the following command:
```

File: pd-recover.md [hunk 11]
```diff
@@ -7,42 +7,152 @@ aliases: ['/docs/dev/reference/tools/pd-recover/']
...
+
+```
+10.0.1.15 | CHANGED | rc=0 >>
+2019/10/14 19:23:04.688 client.go:161: [info] [pd] init cluster id 6747551640615446306
+……
+```
+
+#### Get `[info] PD cluster` from TiKV log
+
+To get the `[info]` PD cluster from the TiKV log, run the following command:
```
Comment (TomShawn) [inline pd-recover.md:54]:
```suggestion
To get the `[info] PD cluster` from the TiKV log, run the following command:
```

**Comment by yikeke** [comment]
Please resolve conflicts. @ran-huang

**Comment by ran-huang** [comment]
@TomShawn PTAL~

**Comment by TomShawn** [comment]
lgtm

**Pull request merged**